app-modules/service-management/database/migrations/2023_09_01_143524_create_service_request_types_table.php

```diff
@@ -45,6 +45,8 @@ public function up(): void
             $table->uuid('id')->primary();
 
             $table->string('name');
+            $table->text('description')->nullable();
+            $table->string('icon')->nullable();
```

Comment:
This will need to be added in a separate migration. Including a `down` method and we will need to ensure that the app will not break if the code is deployed before the migration is run.